tasks/databases.yml

```diff
@@ -1,6 +1,8 @@
 # file: postgresql/tasks/databases.yml
 
 - name: PostgreSQL | Make sure the PostgreSQL databases are present
+  sudo: yes
+  sudo_user: "{{postgresql_admin_user}}"
```

Comment:
Wont this approach fail if your database is on a different server? had you tried `login_host` and `login_password` already?... that said i can see that `login_host` is missing from below, but strangely not from user creation